geometry_msgs/Pose start_point
geometry_msgs/Pose goal
---
nav_msgs/Path Path